Tasks/Gradle/CodeAnalysis/pmd.gradle

```diff
@@ -0,0 +1,31 @@
+rootProject {
```

Comment:
This file kicks off the PMD analysis. In Gradle, applying the 'pmd' plugin will automatically add 'pmdMain' and 'pmdTest' tasks. These become part of the 'build' task.